### DISCUSSION TITLE
How do I route Synology DSM/Drive/Photos through Traefik?

### DISCUSSION THREAD

**Discussion by wiffleman** [htpcBeginner/docker-traefik#238] [Q&A] [answered]
As per title. Looking to route into dsm/drive/photos via [app].domain.com but am unsure how to do this?

Thanks!

**Comment by Cartache** [comment, marked as answer]
You should look into the appdata/traefik2/rules folder. You have there examples to forward the request to another server or a non docker service. But you will need to have a basic functional docker-compose.
